catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -119,7 +123,20 @@ public void init() {
         "/geofeature/suggestions",
         (req, res) -> {
           String query = req.queryParams("q");
-          List<Suggestion> results = this.featureService.getSuggestedFeatureNames(query, 10);
+          List<Suggestion> resultSuggestions =
+              this.featureService.getSuggestedFeatureNames(query, 10);
+          Map<String, Object> latLonLiteral = getLatLonLiteral(query);
```

Comment:
If you rename `latLonLiteral` to `latLonSuggestions` it makes it a bit more clear that you are trying to place lat/lon suggestions first in the suggestion list in this snippet of code
```java
          if (!latLons.isEmpty()) {
            results.add(0, latLonLiteral);
          }
```